### PR TITLE
README.md: add some libraries info

### PR DESCRIPTION
**Pull request by willson-chen**
Add some information of libraries depended by GD. Add a note to GD user about the license of libimagequant.

### CODE REVIEW

**Comment by willson-chen** [comment]
> Thanks! Looks basically good to me, but I don't think that libiconv is required; probably any iconv() implementation could be used (e.g. provided by libc).

Yes, you are right. I have deleted that item. :)

File: README.md [hunk 1]
```diff
@@ -45,3 +45,12 @@ It also has optional support for more formats via external libraries:
 * [TIFF](https://en.wikipedia.org/wiki/Tagged_Image_File_Format) via [libtiff](http://www.libtiff.org/)
 * [WebP](https://en.wikipedia.org/wiki/WebP) via [libwebp](https://developers.google.com/speed/webp/)
 * [XPM](https://en.wikipedia.org/wiki/X_PixMap) via [libXpm](http://xorg.freedesktop.org/)
+
+Besides that, GD depends on some external libraries, which are all optional
+and disabled by default:
+
+* [FreeType](https://freetype.org)
```
Comment (vapier) [inline README.md:52]: the entries above tend to explain why the dep is here.  it would help to add a small bit of text explaining the purpose of each one.

**Comment by willson-chen** [comment]
Hi @vapier. I have improved it by adding some description. Could you please review again?

**Comment by vapier** [comment]
thx, lgtm

**Pull request merged**